### PR TITLE
fix select from information_schema tables error

### PR DESCRIPTION
**Pull request by starrocks-xupeng**
for https://github.com/StarRocks/starrocks/issues/278

### CODE REVIEW

File: be/src/exec/vectorized/schema_scan_node.cpp [hunk 1]
```diff
@@ -101,6 +101,9 @@ Status SchemaScanNode::prepare(RuntimeState* state) {
     const std::vector<SlotDescriptor*>& src_slot_descs = _schema_scanner->get_slot_descs();
     const std::vector<SlotDescriptor*>& dest_slot_descs = _dest_tuple_desc->slots();
     int slot_num = dest_slot_descs.size();
+    if (0 == src_slot_descs.size()) {
```
Comment (kangkaisen) [inline be/src/exec/vectorized/schema_scan_node.cpp:104]:
```suggestion
    if (src_slot_descs.empty()) {
```

File: be/src/exec/vectorized/schema_scan_node.cpp [hunk 2]
```diff
@@ -235,10 +238,11 @@ Status SchemaScanNode::get_next(RuntimeState* state, ChunkPtr* chunk, bool* eos)
     if (nullptr == chunk->get()) {
         return Status::InternalError("Failed to allocate new chunk.");
     }
-    // init column information
-    for (auto& slot_desc : _dest_tuple_desc->slots()) {
-        ColumnPtr column = chunk_tmp->get_column_by_slot_id(slot_desc->id());
-        (*chunk)->append_column(std::move(column), slot_desc->id());
+    if (src_slot_descs.size()) {
```
Comment (kangkaisen) [inline be/src/exec/vectorized/schema_scan_node.cpp:241]:
```suggestion
    if (!src_slot_descs.empty ()) {
```

Comment (starrocks-xueli) [reply be/src/exec/vectorized/schema_scan_node.cpp]: done

File: be/src/exec/vectorized/schema_scan_node.cpp (hunk 1 above)
Comment (starrocks-xueli) [reply be/src/exec/vectorized/schema_scan_node.cpp]: done

**Review by kangkaisen — APPROVED** [review]

**Review by stdpain — APPROVED** [review]

**Pull request merged**